lms/templates/seq_module.html

```diff
@@ -4,7 +4,7 @@
 <div id="sequence_${element_id}" class="sequence" data-id="${item_id}" data-position="${position}" data-ajax-url="${ajax_url}" data-next-url="${next_url}" data-prev-url="${prev_url}">
   % if banner_text:
     <div class="pattern-library-shim alert alert-information subsection-header" tabindex="-1">
-      <span class="pattern-library-shim icon alert-icon fa fa-bullhorn" aria-hidden="true"></span>
+      <span class="pattern-library-shim icon alert-icon fa fa-info-circle" aria-hidden="true"></span>
```

Comment:
This meaningful icon needs a text equivalent:  ```<span class="sr">Important!&nbsp;</span>```

Reply:
fixed